bot/engine/src/main/kotlin/engine/user/UserPreferences.kt

```diff
@@ -60,7 +60,16 @@ data class UserPreferences(
     /**
      * The initial locale of the user
      */
-    var initialLocale: Locale = locale
+    var initialLocale: Locale = locale,
+    /**
+     * Full name or screen name of the user in the application.
+     */
+    var userName: String? = null,
```

Comment:
```suggestion
    var username: String? = null,
```
All lowercase seems more standard